src/framework/shortcuts/data/shortcuts-Mac.xml

```diff
@@ -1093,7 +1093,7 @@
     </SC>
   <SC>
     <key>text-b</key>
-    <seq>Ctrl+b</seq>
+    <seq>Ctrl+B</seq>
```

Comment:
lowercase letters don't seem to work here - but actually even though MU3 shows these as customisable shortcut keys, they aren't - if you change the shortcut key combination for this command (likewise text-i & text-u) it's not honoured. Fixed in MU4, they're top-level actions now.